### DISCUSSION TITLE
Tại sao bạn lại phải dùng những link ngẫu nhiên nhỉ

### DISCUSSION THREAD

**Discussion by MagicalDrizzle** [abpvn/abpvn#548] [General]
Kiểu bạn vào đây ấy, thì mình thấy là toàn link tạo ra ngẫu nhiên và cần phải đổi liên tục khi bạn update list
https://abpvn.com/get
Còn dùng thẳng link github thì nó luôn luôn chỉ vảo file mới nhất, không cần phải đổi gì cả
https://raw.githubusercontent.com/abpvn/abpvn/master/filter/abpvn_ublock.txt

**Comment by hoang-rio** [comment]
Link ngẫu nhiên nhằm mục đích thống kê thôi bạn. Không có lý do gì đặc biệt hết. Và link ngẫu nhiên đấy là 1 id có thể dùng mãi chứ không phải mỗi lần update bộ lọc cần get 1 link mới đâu. Cập nhật bộ lọc thì chỉ cần vào tiện ích mở rộng rồi cập nhật như bình thường thôi
